screenpy/director.py

```diff
@@ -2,6 +2,10 @@
 Directors handle the meta information that it takes to run a screenplay. There
 is only one of them, so you'll always have access to the same information.
 """
+from typing import Type, TypeVar
```

Comment:
It wouldn't be a PR review from @perrygoy without the nittiest little pick—`black` doesn't handle this for some reason, but i've been putting a newline after the module docstrings and before the first import. ":D

```python
"""

from typing import Type, TypeVar
```

Reply:
fixed